packages/smooth_app/lib/pages/preferences_v2/roots/contributions_root.dart

```diff
@@ -67,7 +68,12 @@ class ContributionsRoot extends PreferencesRoot {
   ) {
     return PreferenceTile(
       icon: Icons.add_circle_outline,
-      title: appLocalizations.preferences_contributions_new_products_title,
+      title: appLocalizations.preferences_contributions_new_products_title(
+        const LazyCounterUserSearch(
+              UserSearchType.CONTRIBUTOR,
+            ).getLocalCount(context.read<UserPreferences>()) ??
+            0,
+      ),
```

Comment:
```suggestion
      title: appLocalizations.user_search_contributor_title,
```
@darshanhtailor We already have a label that matches perfectly, so please use it instead.

Reply:
Aah okay! This doesn't display the count in title though, is that fine?

Reply:
@darshanhtailor It's fine, as it's consistent:
* the other labels on that page don't display the count in the title either
* the count is still displayed, on the right side, cf. the other buttons on that page
* the page you go to when you click on that button does use that very same label

Reply:
Thanks @monsieurtanuki! Updated the PR.